DatePickerIOS.js

```diff
@@ -26,9 +26,10 @@ const RCTDatePickerIOS = requireNativeComponent('RNDatePicker')
  * source of truth.
  */
 export default class DatePickerIOS extends React.Component {
-  static DefaultProps = {
+  static defaultProps = {
```

Comment:
@henninghall Might be worth noting that this change is to fix a long over-looked bug that is even in the official react-native project, `DefaultProps` was never used as the correct `defaultProps`, which is why not setting `mode` actually was `undefined` and setting it to `"datetime"` produced weird side-effects as noticed in my issue.